package/yast2-online-update.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Thu Jan  8 07:59:39 UTC 2015 - lslezak@suse.cz
+
+- Reporting products that have reached their end of support when
+  Online Update module is started (FATE#300591)
+- 2.17.24
```

Comment:
This will hopefully not break SP3 where we need to start using 2.17.23.X from now on.